src/javaRosa/util.js

```diff
@@ -252,7 +252,8 @@ define([
     var insertOutputRef = function(vellum, target, path, mug, dateFormat) {
         var output = getOutputRef(path, dateFormat),
             form = vellum.data.core.form;
-        if ((!mug && vellum.opts().features.rich_text) || (mug && mug.supportsRichText())) {
+        if ((!mug && form.useRichText !== false) ||
```

Comment:
Is it possible to initialize `form.useRichText` somewhere so we could get away from doing strict equality checks on booleans?